eZ/Publish/Core/FieldType/RichText/Resources/stylesheets/docbook/xhtml5/edit/core.xsl

```diff
@@ -627,6 +627,34 @@
     </xsl:element>
   </xsl:template>
 
+  <!-- Custom template tag code -->
```

Comment:
copy-paste typo I guess, should be `Custom style tag code`, right? (the "template" doesn't refer to XSL markup but to eZ templates which are more widely known as Custom Tags)